hbase_storage/src/main/java/com/splicemachine/si/data/hbase/coprocessor/SIObserver.java

```diff
@@ -404,10 +405,14 @@ protected boolean shouldReadAllVersions(OperationWithAttributes op) {
 
     @SuppressWarnings("RedundantIfStatement") //we keep it this way for clarity
     protected void doesTableNeedSI(TableName tableName){
+
         SConfiguration config = HConfiguration.getConfiguration();
-        TableType tableType= EnvUtils.getTableType(config,tableName);
         spliceTable = tableName.getNamespaceAsString().equals(config.getNamespace());
-
+        if (tableName.getQualifierAsString().equals(HBaseConfiguration.CONGLOMERATE_SI_TABLE_NAME)) {
+            tableEnvMatch = true;
```

Comment:
Is there a reason we need to check for the conglomerate SI table here, but not other tables that use SI like DROPPED_CONGLOMERATES_TABLE_NAME ?

Reply:
The other non-numeric conglomerates are not managed by transaction system. I think DROPPED_CONGLOMERATES_TABLE_NAME does not use SI

Reply:
The other non-numeric conglomerates are not managed by transaction system. I think DROPPED_CONGLOMERATES_TABLE_NAME does not use SI

Reply:
createSITable and createNonSITable are confusing names. They essentially do the same thing. I will clean up some code